packages/react-wildcat-handoff/src/utils/serverRender.js

```diff
@@ -134,7 +134,7 @@ function getHtmlNotFoundTemplate(serverSettings) {
     if (htmlNotFoundTemplate) {
         return {
             status: 404,
-            html: htmlNotFoundTemplate
+            html: htmlNotFoundTemplate()
```

Comment:
I think you need to test that this is a function, codecov report didn't seem to get uploaded